include/deep_mem.h

```
@@ -12,7 +12,7 @@
 #define P_FLAG_OFFSET (1) /* previous block is allocated */
 #define P_FLAG_MASK (0x00000002)
 #define BLOCK_SIZE_MULTIPLIER (3)
-#define BLOCK_SIZE_MASK (0xfffffff4)
+#define BLOCK_SIZE_MASK (0xfffffffc)
```

this bit mask error is root cause?

---

No, but I've checked with Joey; this should be fffffffc

---

done!